src/app/stores/wallet/reducer.ts

```diff
@@ -46,13 +45,13 @@ const initialWalletState: WalletState = {
   seedPhrase: '',
   encryptedSeed: '',
   fiatCurrency: 'USD',
-  btcFiatRate: new BigNumber(0),
-  stxBtcRate: new BigNumber(0),
-  stxBalance: new BigNumber(0),
-  stxAvailableBalance: new BigNumber(0),
-  stxLockedBalance: new BigNumber(0),
+  btcFiatRate: '0',
```

Comment:
If we some how end up with an invalid number, does the app crash while trying to parse to a BigNumber?

Reply:
Nope, we check the btcFiatRate value to ensure it's a valid number before setting it. See the `numberLikeToStringOrThrow` function in the actionCreator.ts file